algorithm/cgal/Grow.h

```diff
@@ -1,96 +1,226 @@
 #pragma once
 
+#include <CGAL/Constrained_Delaunay_triangulation_2.h>
 #include <CGAL/Exact_predicates_exact_constructions_kernel.h>
 #include <CGAL/Polygon_mesh_processing/compute_normal.h>
 #include <CGAL/Polygon_mesh_processing/corefinement.h>
+#include <CGAL/Polygon_mesh_processing/transform.h>
+#include <CGAL/Polygon_triangulation_decomposition_2.h>
 #include <CGAL/Surface_mesh.h>
+#include <CGAL/boost/graph/generators.h>
+#include <CGAL/connect_holes.h>
+#include <CGAL/convex_hull_3.h>
+#include <CGAL/mark_domain_in_triangulation.h>
+#include <CGAL/partition_2.h>
 
 #include "Geometry.h"
+#include "manifold_util.h"
+#include "transform_util.h"
 
-static int Grow(Geometry* geometry, size_t count, bool x, bool y, bool z) {
-  typedef CGAL::Exact_predicates_exact_constructions_kernel EK;
-  typedef CGAL::Surface_mesh<EK::Point_3> Surface_mesh;
-  size_t size = geometry->size();
+static int Grow(Geometry* geometry, size_t count) {
```

Comment:
What does count do here?

Reply:
Count is the number of geometries in the input (as opposed to the tool).

There's probably a better naming choice for that.